numba_dpex/examples/kernel/scan.py

```diff
@@ -7,7 +7,6 @@
 import dpnp as np
 
 import numba_dpex as ndpx
```

Comment:
Better to keep the import statements consistent in all examples. If were doing "import numba_dpex as dpex" in other examples, we should do the same here.

Reply:
Done.